javascript/home.js

```diff
@@ -3,56 +3,74 @@ var portFlag = false;
 var contactFlag = false;
 var aboutFlag = false;
 $(document).ready(function(){
-  $('#portfolio').click(function(){
+  $('#portfolio > h1').click(function(){
     if (portFlag === false){
       $('#portfolio').animate({
         top: '-=100',
-        width: '400'
+        width: '96%'
       }, 1000 );
       portFlag = true;
-    } else {
-      $('#portfolio').animate({
-        top: '+=100',
-        width: '125'
-      }, 1000 );
-      portFlag = false;
+      $('#portfolio > h3').fadeToggle(1000);
+      $('#about').fadeToggle(1000);
+      $('#contact').fadeToggle(1000);
+      $('.portfolioPiece').slideToggle(1000);
     }
+  });
+  $('#portfolio > h3').click(function(){
```

Comment:
I should have caught this earlier. But the .click() is not best practice and you should use .on() I would refactor this.